### ISSUE TITLE
snap broken on amd64 devices

### ISSUE THREAD

**Issue by Bloodiko** [mmtrt/leagueoflegends#8]
Hey there,

i've just tried to update the snap to the newest version (used the 15 before), but now i'm unable to start/install league (and also unable to revert , cause i removed the folder with the 15 version).

I saw that you removed the wineloader64 which might be causing this , but i have no clue how to fix this myself, as the folders are mounted manually by snap.

The main error why i cant run it is the following:
wine: WINEARCH set to win64 but '/home/ebiko/snap/leagueoflegends/common/.wine' is a 32-bit installation.
basically , my snap still wants to launch LOL with wine64 , but as the wineloader64 is removed it thinks its a 32 installation , and does not launch properly (im using 64bit OS) 

except some other smaller errors on the startup from lol (needing to launch several times before it works, and sometimes with the CPU spikes from the launcher (up to maximum cpu usage on all cores))  the snap works really good, like it

will provide further data if needed :+1:

**Comment by mmtrt** [comment]
You should have read issue #7 which stats that on point 4 necessary step to migrate 64bit `WINEPREFIX`  
* Backup game files before removing win32 `WINEPREFIX` --> `$HOME/snap/leagueoflegends/common/.wine`
* Game should be here `$HOME/snap/leagueoflegends/common/.wine/drive_c/Riot Games`
* Backup `Riot Games` then remove `$HOME/snap/leagueoflegends` and now run snap which will install necessary packages when it ask for `Region selection` close that dialog now restore game folder `Riot Games` to `WINEPREFIX` which would be `$HOME/snap/leagueoflegends/common/.wine/drive_c`
* Run snap profit

As for high cpu usage which does happens here as well so best temporary fix is that close the launcher entirely when game loads after champ selection should help in fps as well.
